go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/LeoCBS/httpmiddleware
+
+go 1.17
```

Comment:
You are testing with go 1.20 but the module requires 1.17. Could it have some inconsistency in the future?

Reply:
good point, i will change this value on next PR :) @danielfmelo